assets/constants.ts

```diff
@@ -4956,250 +4915,253 @@ export default {
+    "relatedLanguages": ["kab"]
   }, {
     "id": "ko",
     "description": "Korean",
-    "relatedLanguages": ["ko"],
-    "direction": "ltr"
+    "relatedLanguages": ["ko"]
   }, {
     "id": "lv",
     "description": "Latvian",
-    "relatedLanguages": ["lv"],
-    "direction": "ltr"
+    "relatedLanguages": ["lv"]
   }, {
     "id": "lt",
     "description": "Lithuanian",
-    "relatedLanguages": ["lt"],
-    "direction": "ltr"
+    "relatedLanguages": ["lt"]
   }, {
     "id": "no",
     "description": "Norwegian",
-    "relatedLanguages": ["no"],
-    "direction": "ltr"
+    "relatedLanguages": ["no"]
   }, {
     "id": "fa",
     "description": "Persian",
-    "relatedLanguages": ["fa"],
-    "direction": "rtl"
+    "relatedLanguages": ["fa"]
   }, {
     "id": "pl",
     "description": "Polish",
-    "relatedLanguages": ["pl"],
-    "direction": "ltr"
+    "relatedLanguages": ["pl"]
   }, {
     "id": "pt",
     "description": "Portuguese",
-    "relatedLanguages": ["pt"],
-    "direction": "ltr"
+    "relatedLanguages": ["pt"]
   }, {
     "id": "ps",
     "description": "Pashto",
-    "relatedLanguages": ["ps"],
-    "direction": "rtl"
+    "relatedLanguages": ["ps"]
   }, {
     "id": "ro",
     "description": "Romanian",
-    "relatedLanguages": ["ro"],
-    "direction": "ltr"
+    "relatedLanguages": ["ro"]
   }, {
     "id": "ru",
     "description": "Russian",
-    "relatedLanguages": ["ru"],
-    "direction": "ltr"
+    "relatedLanguages": ["ru"]
   }, {
     "id": "sr",
     "description": "Serbian",
-    "relatedLanguages": ["sr"],
-    "direction": "ltr"
+    "relatedLanguages": ["sr"]
   }, {
     "id": "sk",
     "description": "Slovak",
-    "relatedLanguages": ["sk"],
-    "direction": "ltr"
+    "relatedLanguages": ["sk"]
   }, {
     "id": "sl",
     "description": "Slovenian",
-    "relatedLanguages": ["sl"],
-    "direction": "ltr"
+    "relatedLanguages": ["sl"]
   }, {
     "id": "es",
     "description": "Spanish",
-    "relatedLanguages": ["es"],
-    "direction": "ltr"
+    "relatedLanguages": ["es"]
   }, {
     "id": "sv",
     "description": "Swedish",
-    "relatedLanguages": ["sw"],
-    "direction": "ltr"
+    "relatedLanguages": ["sw"]
   }, {
     "id": "te",
     "description": "Telugu",
-    "relatedLanguages": ["te"],
-    "direction": "ltr"
+    "relatedLanguages": ["te"]
   }, {
     "id": "th",
     "description": "Thai",
-    "relatedLanguages": ["th"],
-    "direction": "ltr"
+    "relatedLanguages": ["th"]
   }, {
     "id": "tr",
     "description": "Turkish",
-    "relatedLanguages": ["tr"],
-    "direction": "ltr"
+    "relatedLanguages": ["tr"]
   }, {
     "id": "uk",
     "description": "Ukrainian",
-    "relatedLanguages": ["uk"],
-    "direction": "ltr"
+    "relatedLanguages": ["uk"]
   }, {
     "id": "vi",
     "description": "Vietnamese",
-    "relatedLanguages": ["vi"],
-    "direction": "ltr"
+    "relatedLanguages": ["vi"]
   }],
 
+  "LANGUAGE_DIRECTIONS": {
```

Comment:
Any specific reason for having this as a separate item? This will make it harder to maintain or add any new language. 

If you want to avoid duplicate code here, then in a separate PR you can combine all the language into one dict and have a separate field/key to specify whether the usage of the Langauge. But in any case all the things related to a lanagueg should be in one place.

Reply:
This was suggested by @sagangwee [see conversation](https://github.com/oppia/oppia/pull/11640#discussion_r555454834).
Should I revert this for now?

Reply:
Just a note -- I chatted with @TheoLipeles about this and I suggest keeping it as @DubeySandeep mentions for now, i.e. with the direction key in the original dict. 

@TheoLipeles has some ideas for a follow-up refactor to reduce duplication more generally. They will file an issue, and I'll defer to them for that follow-up discussion.

Reply:
I reverted this for now, I'll file an issue as well.